autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/drivable_area_expansion.param.yaml

```
@@ -19,13 +19,15 @@
         extra_wheelbase: 0.0 # [m] extra length to add to the wheelbase
         extra_front_overhang: 0.5 # [m] extra length to add to the front overhang
         extra_width: 1.0 # [m] extra length to add to the width
-      dynamic_objects:
-        avoid: false # if true, the drivable area is not expanded in the predicted path of dynamic objects
-        extra_footprint_offset:
-          front: 0.5 # [m] extra length to add to the front of the dynamic object footprint
-          rear: 0.5 # [m] extra length to add to the rear of the dynamic object footprint
-          left: 0.5 # [m] extra length to add to the left of the dynamic object footprint
-          right: 0.5 # [m] extra length to add to the rear of the dynamic object footprint
+      object_exclusion:
+        exclude_static: true # if true, the drivable area is not expanded over static objects
```

```suggestion
        exclude_static: false # if true, the drivable area is not expanded over static objects
```
This feature hasn't ever been used with bus type vehicle, I guess. So, I think it's better to enable this after experiment with real vehicle. What do you think?